Modules/Loadable/Sequences/qSlicerSequencesModule.cxx

```diff
@@ -105,8 +106,8 @@ void qSlicerSequencesModulePrivate::addToolBar()
 
   this->ToolBar->setWindowTitle("Sequence browser");
   this->ToolBar->setObjectName("SequenceBrowserToolBar");
-  mainWindow->addToolBar(this->ToolBar);
-  this->SequenceBrowserModuleOwnsToolBar = false;
+  mainWindow->addToolBar(Qt::BottomToolBarArea, this->ToolBar);
```

Comment:
Bottom toolbar area would be below the slice views? What about as a second row in the top toolbar area? I know you can already drag toolbars to create a second row.

Reply:
Initially I wanted to put the toolbar in the second row of the top area, but when I saw that you cannot set a preferred row but you have to add a toolbar break then I assumed that it would interfere with manual arrangement and automatic save/restore of toolbar positions. Placing it in the bottom  was easy and looked not nice.

After reading your comment, I've tested the toolbar break behavior and it worked perfectly, too.

Considering that placing the sequence toolbar can in a second row is a smaller change visually and that we will have to add more toolbars (for markups placement and editing, favorite layouts, etc.) it makes more sense to add all the toolbars to the top area by default. I'll update the code accordingly.